libs/coin-modules/coin-solana/package.json

```diff
@@ -53,8 +53,8 @@
     "@ledgerhq/logs": "workspace:^",
     "@ledgerhq/types-cryptoassets": "workspace:^",
     "@ledgerhq/types-live": "workspace:^",
-    "@solana/spl-token": "^0.3.7",
-    "@solana/web3.js": "1.77.3",
+    "@solana/spl-token": "^0.4.8",
+    "@solana/web3.js": "1.95.2",
```

Comment:
Same as the other not sure about bumpimg solana deps, is this related to some types changes ?